middleware/fileserver.go

```diff
@@ -128,6 +129,10 @@ func (fh *fileHandler) serveFile(w http.ResponseWriter, r *http.Request, name st
 		return http.StatusNotFound, nil
 	}
 
+	// Add ETag header
+	e := fmt.Sprintf("\"%x-%x\"", d.ModTime().Unix(), d.Size())
+	w.Header().Set("Etag", e)
```

Comment:
Please change this to "ETag", a la:

https://en.wikipedia.org/wiki/HTTP_ETag
https://tools.ietf.org/html/rfc7232#section-2.3

Yes, headers are case insensitive, but we should be strict in what we send, and liberal in what we accept.  :)